src/TitleDetails/AddPieceModal/AddPieceModalContainer.js

```diff
@@ -46,13 +46,22 @@ const AddPieceModalContainer = ({
     ? PIECE_FORMAT_OPTIONS.filter(({ value }) => [PIECE_FORMAT.electronic, PIECE_FORMAT.physical].includes(value))
     : PIECE_FORMAT_OPTIONS.filter(({ value }) => value === initialValues.format);
 
+  const initialCheckboxValues = useMemo(() => (
```

Comment:
FYI interest, useMemo here is not really required as you do `{ ...initialValues, ...initialCheckboxValues }`, with memo only only optimize object calculation and it's not big calculation